JeffShepherd/fitness-tracker
# Feature/final styling

## What does this PR do?

- Adds final position and style to all aspects of the dashboard
- Adds responsiveness to screen sizes 1000px and above (default layout is mobile)

## How should it be tested in the UI?

- Start application in browser with 'npm start'
- Ensure the layout looks good for small and medium/large screen sizes
- Run 'npm test' to ensure all tests still pass
